src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -906,6 +907,10 @@ namespace ReportSizingManager {
 									TotCapTempModFac = CurveValue ( DataTotCapCurveIndex, CoilInWetBulb, OutTemp );
 								} else {
 									TotCapTempModFac = 1.0;
+									
+									if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
+										EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, TotCapTempModFac);
```

Comment:
@mjwitte @rraustad During the coil sizing step, the existing VRF coil uses a DataTotCapCurve to calculate TotCapTempModFac. The new coil type doesn't have such a curve, so I deveoped the DXCoils::CalcVRFCoilCapModFac method to calculate TotCapTempModFac. Do you have any advice?